shell.nix

```diff
@@ -1,11 +1,11 @@
-# A very simple shell.nix file for setting up necessary build tools. This is
-# likely going to be updated using the iohk-specific nixpkgs and a haskel.nix
-# derivation of our cabal.project.
+# A shell setup providing build tools and utilities for a development
+# environment. This is now based on haskell.nix and it's haskell-nix.project
+# (see 'default.nix').
 { compiler ? "ghc8104"
   # Latest haskell.nix for more likely cache hits
 , haskellNix ? import
     (builtins.fetchTarball
-      "https://github.com/input-output-hk/haskell.nix/archive/abb289ff961fb1bd98eb0340088c432f15109807.tar.gz")
+      "https://github.com/input-output-hk/haskell.nix/archive/532e71470b41fc0fd0d3d858ea98d7f07f37d309.tar.gz")
```

Comment:
Do we need to repeat the dependency here? Seems like it's also in `default.nix` and I wonder if that could not bite us should we forget to change one and not the other.

Reply:
I'll factor it out into a make-shift `sources.nix` or so later